docs/user-guide/patterns/sums.md

```diff
@@ -1,14 +1,29 @@
-Tracking Sums
-=============
+# Tracking Sums
 
-```{todo}
-Finish
+## Enforcing Sum of Two Balances Constraint
+```cvl
+invariant directSumOfTwo(address a, address b)
+    (a != b) => (balanceOf(a) + balanceOf(b) <= to_mathint(totalSupply()));
 ```
 
-```{todo}
-Sum of two values is ≤ total
-```
+Ensure that the sum of balances for any two distinct addresses, `a` and `b`, does not exceed 
+the total supply.
+
+## Maintaining Equality Between Sum of Balances and Total Supply
+```cvl
+ghost mathint sumBalances {
+    init_state axiom sumBalances == 0;
+}
 
-```{todo}
-Sum is sum of all values
+hook Sstore balanceOf[KEY address user] uint256 newBalance (uint256 oldBalance) STORAGE
+{
+    sumBalances = sumBalances + newBalance - oldBalance;
```

Comment:
Perhaps comment that ghost variables do not accept `+=` operator?

Reply:
Done